controllers/goharbor/harbor/notarysigner.go

```diff
@@ -82,7 +82,7 @@ func (r *Reconciler) GetNotarySignerCertificateAuthority(ctx context.Context, ha
 			Duration: &metav1.Duration{
 				Duration: duration,
 			},
-			CommonName: r.NormalizeName(ctx, harbor.GetName(), controllers.NotarySigner.String()),
+			CommonName: r.NormalizeName(ctx, harbor.GetName(), controllers.NotarySigner.String(), "CA"),
```

Comment:
How does this fix cert-manager v1.4 compatibility?

Reply:
cert-manger 1.4 doesn't allow using same CN between Cert and CA, so I added a suffix to distinguish them